sycl/test-e2e/Matrix/SPVCooperativeMatrix/SG32/joint_matrix_bf16_fill_k_cache.cpp

```diff
@@ -0,0 +1,24 @@
+//==--- joint_matrix_bf16_fill_k_cache.cpp  - DPC++ joint_matrix----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
+// UNSUPPORTED: gpu-intel-dg2
+// REQUIRES: gpu, aspect-ext_intel_matrix
+// REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
+
+// RUN: %{build} -D__SPIRV_USE_COOPERATIVE_MATRIX -o %t.out -ffp-model=precise
```

Comment:
Also, it seems you are using outdated set of tests. For example, please compare this file with: https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Matrix/SG32/joint_matrix_bf16_fill_k_cache.cpp

Reply:
Fixed